README.md

```diff
@@ -1,66 +1,75 @@
-Crane predictor fetches historical metric data for the monitoring system, such as Prometheus, and identifies the time series that are predictable, for example, system CPU load, memory footprint, application's user traffic, etc. Then it outputs the prediction results, which can be consumed by other crane components, like [Effective HorizontalPodAutoscaler](#effective-horizontalpodautoscaler) and [Analysis as a Service](#analysis-as-a-service). It's also straightforward to apply the prediction results in user applications.
+Crane predictor fetches metric data, and then outputs the prediction results.
+The prediction result can be consumed by other crane components, like [EHPA](#effective-horizontalpodautoscaler) and [Analysis](#analysis-as-a-service).
 
 Please see [this document](./docs/tutorials/using-time-series-prediction.md) to learn more.
 
 ### Effective HorizontalPodAutoscaler
 
 EffectiveHorizontalPodAutoscaler helps you manage application scaling in an easy way. It is compatible with native [HorizontalPodAutoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/) but extends more features.
-EffectiveHorizontalPodAutoscaler supports prediction-driven autoscaling that supported by [TimeSeriesPrediction](#time-series-prediction). Users can forecast the incoming peak flow and scale up their application ahead, also users can know when the peak flow will end and scale down their application gracefully. Besides, EffectiveHorizontalPodAutoscaler also defines several scale strategies to support different scaling scenarios.
-
-- **Reliability**: Guarantee both scalability and availability
-- **Responsiveness**: Scale-up fast enough to successfully handle the increase in workload
-- **Observability**: Support Preview mod and automatic observe replicas 
 
 Please see [this document](./docs/tutorials/using-effective-hpa-to-scaling-with-effectiveness.md) to learn more.
 
 ### Analysis as a Service
 
-Analysis Service gives you recommendations about cost optimize. It scans your cluster resources such as Deployment, StatefulSet and provides a variety of strategies to analyze the resource then recommend how to optimize it. Analysis and Recommendation are CustomResourceDefinition that can integrate with your own systems.
+Analysis Service gives you recommendations about cost optimize.
 
-Here we provide two Analysis Services:
+Two Recommendations are currently supported:
 - **ResourceRecommend**: Recommend container requests & limit resources based on historic metrics.
 - **Effective HPARecommend**: Recommend which workloads are suitable for autoscaling and provide optimized configurations such as minReplicas, maxReplicas.
 
-### Application Ensurance
+### QoS Ensurance
+
+## Repositories
+
+Crane is composed of the following components:
+- [craned](cmd/craned). - main crane control plane.
+  - **Predictor** - Predicts resources metrics trends based on historical data
+  - **AnalyticsController** - Analyzes workloads for wastes
```

Comment:
- **AnalyticsController** - Analyzes resources and generate related recommendations.